documentation/deployment.md

```diff
@@ -7,6 +7,21 @@ The CEP is a standard Node.js app and doesn't require more dependencies than the
 
 A mongoDB 2.4 database should be working and accesible before perseo can be started
 
+### Installation using Docker
+
+The last development version is uploaded as a Docker image to Docker Hub for every PR merged into the `develop` branch.
+Perseo FE needs some components to be present event to be started. Those components can be configured using:
+
+* Environment variables, as in the following example:
+```
+docker run -e "PERSEO_MONGO_HOST=52.26.28.74" -e "PERSEO_CORE_URL=http://52.26.93.210:8080" fiwareiotplatform/perseocore
```

Comment:
Not sure if we should include public IPs in public documents (even when they belong to a very well known infrastructure provider, see http://whatmyip.co/view/ip_addresses/874126336/52.26.28.0_52.26.28.255)